streaming/stream_session.cc

```diff
@@ -214,7 +214,7 @@ void stream_session::init_messaging_service_handler() {
                         });
                     };
                     //FIXME: discarded future.
-                    (void)mutation_writer::distribute_reader_and_consume_on_shards(s, dht::global_partitioner(),
+                    (void)mutation_writer::distribute_reader_and_consume_on_shards(s, s->get_partitioner(),
```

Comment:
Again, a function which takes both a schema and a partitioner no longer needs to and can take just a schema. But I guess we can clean this up later.

Reply:
Thanks - fixed.